.github/workflows/update-catalog.yaml

```diff
@@ -0,0 +1,39 @@
+name: Generate FBC catalogs
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+    branches:
+      - main
+      - development
+    paths:
+      - 'catalog/**'
+
+jobs:
+  update-catalog:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Install OPM
+        run: |
+          wget https://github.com/operator-framework/operator-registry/releases/download/v1.53.0/linux-amd64-opm
+          chmod +x linux-amd64-opm
+          mkdir .bin
+          mv linux-amd64-opm ./.bin/opm
+          echo "$GITHUB_WORKSPACE/.bin" >> $GITHUB_PATH
+
+      - name: Log into registry.redhat.io
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          registry: registry.redhat.io
+          username: ${{ secrets.RH_REGISTRY_USERNAME }}
+          password: ${{ secrets.RH_REGISTRY_TOKEN }}
+
+      - name: Generate catalog
+        run: ./scripts/update-catalog-opm.sh opentelemetry
+
+      - uses: stefanzweifel/git-auto-commit-action@b863ae1933cb653a53c021fe36dbb774e1fb9403
```

Comment:
For security reasons, I'd avoid 3rd-party github actions entirely. Can we use regular git commands like in the operator repo?: https://github.com/grafana/tempo-operator/blob/main/.github/workflows/prepare-release.yaml#L62-L67

Reply:
updated